hooks/hiero_get_shot.py

```diff
@@ -20,14 +20,22 @@ def execute(self, task, item, data, **kwargs):
         Takes a hiero.core.TrackItem as input and returns a data dictionary for
         the shot to update the cut info for.
         """
-        # get the parent sequence for the Shot
-        sequence = self._get_sequence(item, data)
+
+        # get the parent entity for the Shot
+        parent = self.parent.execute_hook(
+            "hook_get_shot_parent",
+            hiero_sequence=item.parentSequence(),
+            data=data
+        )
```

Comment:
@manneohrstrom Is there any way to switch to new style hooks in a way that would let us transition to having get_parent as another method in this class in a way that wouldn't break for people who've taken over this hook?  This app already has a ton of hooks... kind of sucks to introduce another one and if anybody changed the parenting behavior for shots they're going to have to override the new hook anyway.